omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6.rst

```diff
@@ -96,17 +86,67 @@ Configure OMERO.web and create the NGINX OMERO configuration file::
     omero config set omero.web.application_server wsgi-tcp
     omero web config nginx --http "${WEBPORT}" --servername "${WEBSERVER_NAME}" > /opt/omero/web/omero-web/nginx.conf.tmp
 
-For more customization, please read :ref:`customizing_your_omero_web_installation`.
+OMERO.web offers a number of configuration options.
+    The configuration changes **will not be applied** until
+    Gunicorn is restarted using ``omero web restart``.
+
+    -  Session engine:
+
+      -  OMERO.web offers alternative session backends to automatically delete stale data using the cache session store backend, see :djangodoc:`Django cached session documentation <topics/http/sessions/#using-cached-sessions>` for more details.
+
+      - `Redis <https://redis.io/>`_ requires `django-redis <https://github.com/jazzband/django-redis/>`_ in order to be used with OMERO.web. We assume that Redis has already been installed. Follow the step-by-step deployment guides from :doc:`../web-deployment`. To configure the cache, run::
+
+          $ omero config set omero.web.caches '{"default": {"BACKEND": "django_redis.cache.
+          RedisCache", "LOCATION": "redis://127.0.0.1:6379/0"}}'
+
+      -  After installing all the cache prerequisites set the following::
+
+          $ omero config set omero.web.session_engine django.contrib.sessions.backends.cache
+
+
+    - Use a prefix:
+
+      By default OMERO.web expects to be run from the root URL of the webserver.
+      This can be changed by setting :property:`omero.web.prefix` and
+      :property:`omero.web.static_url`. For example, to make OMERO.web appear at
+      `http://example.org/omero/`::
+
+          $ omero config set omero.web.prefix '/omero'
+          $ omero config set omero.web.static_url '/omero/static/'
+
+      and regenerate your webserver configuration.
+
+    - Use a different host:
```

Comment:
Is this something people are likely to do? It requires copying all the OMERO.web static files to the separate Nginx server so sounds like an advanced customisation option.

Reply:
It is certainly advanced. Trying to combine all the options instead of being pushed to several pages as it was before.

Reply:
Alternatively delete it?

Reply:
again that was in the original doc. I can label it as advanced

Reply:
The impression I get is this is meant to be a walkthrough with information on configurating properties that users are likely to want to change. If it's advanced I think it's better to leave it out of this doc. E.g. remove it completely, move it to a separate advanced page (shared amongst all walkthroughs), or move the documentation into https://github.com/ome/omero-web/blob/ec07fe59bf3af6ae002d634675d59a4b0b679b18/omeroweb/settings.py#L362-L366 so it's self contained?

Reply:
moving the doc to settings is probably the most suitable option